rpm/qm.spec

```diff
@@ -5,7 +5,6 @@
 %global moduletype services
 %global modulenames qm
 %global seccomp_json /usr/share/%{modulenames}/seccomp.json
-%global setup_tool %{_prefix}/share/%{modulenames}/setup
```

Comment:
I would use it here, 
%if %{include_setup} == "true"
setup_tool %{_prefix}/share/%{modulenames}/setup
%endif

And return the deleted file in Makefile

Reply:
I will add a patch informing setup is not the official and supported way to deploy QM for automotive as @alexlarsson shared. Will be much more simple but we can always improve on the road with `if's/env vars or subpackages` thanks @Yarboa